.husky/pre-commit

```diff
@@ -2,3 +2,5 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+npm run test
```

Comment:
Our goal is to add unit test checking for the new pull request, so we need to add it into our github action.